jdaviz/app.py

```diff
@@ -311,16 +311,31 @@ def _link_new_data(self):
 
         # Can't link if there's no world_component_ids
         wc_new = self.data_collection[-1].world_component_ids
-        if wc_new == []:
```

Comment:
This was added in #435 by @rosteen to improve performance. Can we be sure that performance will not degrade by removing this line?

Reply:
The problem is that for moment map data, `world_component_ids` is empty, since it seems like it does not receive spatial WCS infomation ([see related comment](https://github.com/spacetelescope/jdaviz/blob/main/jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py#L31)). Otherwise I could keep that check in.

Reply:
Is it more correct, then, to pass that WCS info to moment output instead of all this code?